.github/workflows/tests_primary.yml

```diff
@@ -245,3 +221,16 @@ jobs:
       if: always()
       shell: bash
 
+  write-pull-request-number:
```

Comment:
It would be ideal to make this part of the action, so that users would not need this additional job.